CONTRIBUTING.md

```diff
@@ -1,27 +1,54 @@
 # How to contribute?
 
-Fork and pull request! 😃
+**Simple way**: Fork and pull request! 😃
 
-NOTE: If the project does not have an issue for what you want to do, create an issue first.
+**NOTE**: If the project does not have an issue for what you want to do, create an issue first.
 
 Feel free to submit [Github Issues](https://github.com/libresign/libresign/issues) or pull requests.
 
-The documentation website is build from the docs folder with vuepress.
+But... follow the complete way:
 
 ## To start front and backend development environment
 
-```bash
-make serve
-```
+* Start the Nextcloud development.
+  > **NOTE**: I suggest to use this: https://github.com/LibreCodeCoop/nextcloud-docker-development/ only because is the environment that I use 😅. I also suggest to use https://github.com/juliushaertl/nextcloud-docker-dev but the setup will be a bit different of this tutorial.
+* Wait to be possible access `localhost` in your browser
+* Inside the folder `volumes/nextloud/apps-extra` clone the LibreSign repository
+* open bash in nextcloud container with `docker compose exec -u www-data nextcloud bash`
```

Comment:
Depending on how the transfer is carried out, the user may not be able to execute the command on line 17, so I think it may be useful to make the command ```cd nextcloud-docker-development``` explicit (if the user is clear using **nextcloud-docker- development**)